tests/test_singularity_containers.py

```diff
@@ -0,0 +1,33 @@
+#from spikeinterface.extractors import toy_example
```

Comment:
Ben, I would move this file to spikeinterface/sorters/tests.
We choose to put testing file "inplace" in the relative of the submodule.

Reply:
That's fine as long as we can keep this ci here